### PR TITLE
add pr template

### PR DESCRIPTION
**Pull request by CJ-Wright**
@conda-forge/staged-recipes @conda-forge/core please review/merge when you get a chance!

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!-- 
+Thank you very much for putting in this recipe PR!
+
+If you need help or review please let us know.
+There are langauge specific teams for reviewing recipes
```
Comment (ocefpaf) [inline .github/pull_request_template.md]: language

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!--
```
Comment (ocefpaf) [inline .github/pull_request_template.md]: Do we need the `<!-- ` and `-->`?

Comment (CJ-Wright) [reply .github/pull_request_template.md]: I can remove them, but then the template would show up in the actual PR body.

Comment (ocefpaf) [reply .github/pull_request_template.md]: No problem. I don't really know that is the standard behavior for things like that. I guess that we should leave it commented out.

File: .github/pull_request_template.md (hunk 1 above)
Comment (CJ-Wright) [reply .github/pull_request_template.md]: I though we were all working on the newly discovered lan gauge boson. Am I in the wrong place? :laughing:

File: .github/pull_request_template.md (hunk 2 above)
Comment (CJ-Wright) [reply .github/pull_request_template.md]: I was mostly going off of https://raw.githubusercontent.com/conda/conda/master/.github/ISSUE_TEMPLATE.md

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Currently available teams are:
+- python `@conda-forge/help-python`
+- r `@conda-forge/help-r`
+- java `@conda-forge/help-java`
+- nodejs `@conda-forge/help-nodejs`
+- c/c++ `@conda-forge/help-c-cpp`
+- perl `@conda-forge/help-perl`
+- Julia `@conda-forge/help-julia`
+- python/c hybrid `@conda-forge/help-python-c`
```
Comment (dougalsutherland) [inline .github/pull_request_template.md]: Maybe move up to be next to python?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<!-- 
+Thank you very much for putting in this recipe PR!
+
+If you need help or review please let us know.
```
Comment (dougalsutherland) [inline .github/pull_request_template.md]: Maybe something like: "This repository is very active, so if you need help with a PR or once it's ready for review, please let the right people know."

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+There are language specific teams for reviewing recipes.
```
Comment (dougalsutherland) [inline .github/pull_request_template.md]: Nit: "language-specific".

**Comment by jakirkham** [comment]
Thanks for doing this @CJ-Wright. This looks pretty good to me. Do we have people in all of the teams listed?

**Comment by CJ-Wright** [comment]
Yes, but some of the teams are just me so far. (Which needs to be fixed, I'd like to remove myself from of the teams, since the only Julia recipes I know are from Julia Child :smile:) I'm still waiting on the majority of staged to respond to the issue. Core should be able to add themselves, but I think staged needs to let me know one way or another (they can always request membership too).

**Comment by dougalsutherland** [comment]
@CJ-Wright I wasn't able to add myself (at least by just clicking on it, maybe there's some back-door path to do it), so maybe only org admins and not all of core can add themselves.

**Comment by jakirkham** [comment]
I think I fixed this @dougalsutherland. Please recheck.

**Comment by dougalsutherland** [comment]
It looks like you can only at-mention a team if you're a member of the organization:



This seems like a problem for telling people to at-mention teams. :(

(https://github.com/conda-forge/staged-recipes/pull/5348#issuecomment-375045227)

**Comment by dougalsutherland** [comment]
We could have a webservice automatically add anyone who sends a PR to the org? Or one to look for comments that try to mention people but aren't on the team, and mention them itself?

**Comment by CJ-Wright** [comment]
Did we always have this problem? (eg before I added subteams?)

**Comment by dougalsutherland** [comment]
Yes, I think so. We just haven't explicitly been telling people to at-mention teams for very long, so the people who knew to do it were mostly already in the org.

**Comment by CJ-Wright** [comment]
Whew, I thought I had broken things massively :relieved:

**Comment by CJ-Wright** [comment]
Maybe we add them as outside collaborators? That might give them @mention privlages

File: .github/pull_request_template.md (hunk 2 above)
Comment (CJ-Wright) [reply .github/pull_request_template.md]: That is fair, although maybe out of scope for this PR?

**Comment by CJ-Wright** [comment]
We could also have a bot which when pinged pings staged (although this really sounds like a GH driven issue)

**Comment by jakirkham** [comment]
Interesting, thanks for looking into this all.

> We could have a webservice automatically add anyone who sends a PR to the org? Or one to look for comments that try to mention people but aren't on the team, and mention them itself?

Maybe we can consider a special case application of the proposal in issue ( https://github.com/conda-forge/conda-forge-webservices/issues/164 ).

**Comment by jakirkham** [comment]
There's the `@conda-forge/all-members` team, which has basically no permissions, but are in the org.

**Comment by CJ-Wright** [comment]
Should I add a note that this only applies to people who are already maintainers or is this a blocking issue?

**Comment by jakirkham** [comment]
I don't think it is blocking. We have the same issue with the `staged-recipes` team now, right? So this issue just extends to specific help teams. If anything, this will provider further incentive to solve the problem if we go ahead.

**Comment by jakirkham** [comment]
Thoughts on adding OS help teams?

**Comment by dougalsutherland** [comment]
OS teams sound reasonable.

I can try maybe this weekend to add a webservices bot that listens for attempted pings of these teams (or any conda-forge team?) by non-members, and then re-pings them + adds the person who tried it to the org.

**Comment by CJ-Wright** [comment]
Can we move the OS teams to a different PR?

**Comment by jakirkham** [comment]
Sure. Should we be merging this?

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes poke

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes poke

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**